src/api/utils/index.ts

```diff
@@ -13,6 +13,12 @@ export const windowLoaded = new Promise((accept, reject) => {
   }, false)
 })
 
+export const readFileUpload = (file: any) =>
```

Comment:
Using native types this can be `export const readFileUpload = (file: File): Promise<ArrayBuffer> =>`

Reply:
done